### ISSUE TITLE
Mucus plugging

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7176]
**Preferred term label:**
Mucus plugging

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6026239/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

619466 

PMID: 34077761  3/7
**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Proposed definition

Definition: Mucus that is abnormally thick in consistency (ie, inspissated) and plugs the airway is known as a mucus plug. Mucus plugs can partially or completely obstruct one or more airways and cause serious consequences, including atelectasis and recurrent infection. 
Comment: Impacted mucus can also produce a bronchial cast, which is a semisolid occlusion within a bronchus that takes the shape of the airway within which it formed. This type of mucus plug can produce a mass-like appearance on chest imaging that has been termed mucoid pseudotumor. Mucus plugs that are apparent on imaging and on bronchoscopy can constitute diagnostic clues in certain airway diseases. 

@mgriese Is this accurate?
I am uncertain as to the best parent term --  [Abnormal bronchus morphology HP:0025426](https://hpo.jax.org/app/browse/term/HP:0025426) might be a possibility?

**Comment by mgriese** [comment]
Mucus plug
is a term which could be listed under abnormal morphology, as one can see it. 
Definition above is OK, I would include that they consist of mucins and cells;
the comment however I would delete. Mucoid pseudotumor (I hear it for the first time) is unusual and should be defined as a term on its own, if wished to include. 
Airway casts are mostly generated by other mechanisms than agglomeration of mucus plugs. See below.

Mucus plugging
is a term I would list under abnormal physiology, as it is a consequence of abnormal mucus clearance (https://hpo.jax.org/app/browse/term/HP:0031602) and accumulation of mucus in the aiways with consecutive obstruction. 

Airway casts 
are replicates of airways from accumulated and condensed mucous and cellular material. These semisolid occlusions take the shape of the airway within they are formed, leading to obstruction of the tracheobronchial tree with consecutive respiratory insufficiency. Based on their principal composition, one can differentiate fibrin casts and mucin casts. Plastic bronchitis is the name of the condition characterized by the formation of airway casts.

Fibrin casts (synonyms pseudomembranes, diphtheroid membranes, type I casts, cellular casts) 
are made of plasma components and inflammatory cells

Mucin casts (synonyms type II casts, acellular casts) 
are mainly made of impacted mucus

Comment: Casts are frequently (4 – 16%) observed in patients with congenital heart disease who have undergone Fontan palliation (PMID: 34713989). Casts are infrequently observed in sickle cell disease‐and acute chest syndrome, airway infections, in particular H1N1 influenza, severe asthma, or after smoke/chemical inhalation injuries

**Comment by pnrobinson** [comment]
thanks, adding
